### PR TITLE
Add mobile key MOBILE_WALLET_REQUIRED

### PR DESCRIPTION
**Pull request by ruixhuang**
Screenshot_20240301_190621_dYdX Debug

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,10 +1,18 @@
 {
   "name": "@dydxprotocol/v4-localization",
-  "version": "1.1.47",
-  "lockfileVersion": 1,
+  "version": "1.1.48",
+  "lockfileVersion": 3,
```
Comment (ruixhuang) [inline package-lock.json:4]: @rosepuppy   This is generated by new version of npm.

`ruihuang@Rh-dydx-mbp v4-localization % npm --version
10.2.4
`
Shouldn't matter, right?

Comment (rosepuppy) [reply package-lock.json:4]: Yea should be good

File: scripts/generated/app.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // !! GENERATED FILE - DO NOT EDIT
```
Comment (rosepuppy) [inline scripts/generated/app.ts:1]: I don't see `MOBILE_WALLET_REQUIRED` added to this file 🤔

Comment (rosepuppy) [reply scripts/generated/app.ts:1]: oh nvm looks like not needed for mobile

**Review by rosepuppy — APPROVED** [review]

**Pull request merged**